### PR TITLE
Matrix VJP to use fused matmuls. (transpose doesn't trace).

### PR DESCRIPTION
**Pull request by pschuh**

### CODE REVIEW

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Review by dan-zheng — APPROVED** [review]

File: stdlib/public/TensorFlow/Gradients.swift [hunk 1]
```diff
@@ -484,7 +484,7 @@ func _vjpMatmul<Scalar : TensorFlowFloatingPoint>(
 ) -> (Tensor<Scalar>, (Tensor<Scalar>) -> (Tensor<Scalar>, Tensor<Scalar>)) {
   let value = matmul(lhs, rhs)
   return (value, { v in 
-    return (matmul(v, rhs.transposed()), matmul(lhs.transposed(), v))
+    return (Raw.matMul(v, rhs, transposeB: true), Raw.matMul(lhs, v, transposeA: true))
```
Comment (rxwei) [inline stdlib/public/TensorFlow/Gradients.swift]: over 80 columns.

**Comment by rxwei** [comment]
it's more efficient anyway, so +1 for using raw.

**Comment by rxwei** [comment]
do you need this merged by v0.3?

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Pull request closed without merging**